packages/api/src/env.js

```diff
@@ -11,9 +13,13 @@ export function envAll (_, env) {
   env.magic = new Magic(env.MAGIC_SECRET_KEY || MAGIC_SECRET_KEY)
 
   env.db = new DBClient({
-    endpoint: env.FAUNA_ENDPOINT || FAUNA_ENDPOINT,
+    endpoint: env.FAUNA_ENDPOINT || FAUNA_ENDPOINT || FAUNA_DEFAULT_ENDPOINT,
```

Comment:
This is optional, but you can't access the value of an undefined global so you need to type check it first.

```suggestion
    endpoint: env.FAUNA_ENDPOINT || (typeof FAUNA_ENDPOINT === 'undefined' ? undefined : FAUNA_ENDPOINT)
```

<img width="479" alt="Screenshot 2021-07-07 at 11 50 02" src="https://user-images.githubusercontent.com/152863/124747092-80063900-df19-11eb-81eb-821edf74f56f.png">